platform/mellanox/mlnx-platform-api/sonic_platform/sfp.py

```diff
@@ -30,7 +30,7 @@
 
 # definitions of the offset and width for values in XCVR info eeprom
 XCVR_INTFACE_BULK_OFFSET = 0
-XCVR_INTFACE_BULK_WIDTH_QSFP = 20
+XCVR_INTFACE_BULK_WIDTH_QSFP = 65
```

Comment:
does this cover both QSFP and QSFP-DD without any degradation?

Reply:
yes, this change will affect only QSFP cable types.
QSFP-DD cables use a different field 'Application Advertisement"